theme/acquia_lift.wizard.theme.inc

```diff
@@ -45,6 +46,9 @@ function theme_acquia_lift_card($variables) {
   }
   $html .= '</div>';
   $html .= '<div class="el-card__content">';
+  if (!empty($card_flag)) {
```

Comment:
I'm going to have to make this flag appear dynamically in response to dragon drop.  Seems like we could either have the flag HTML structure always there and just hidden when unused by CSS or... I could pass the flag value that I intend to activate-deactivate through the theme function and then toggle it via dragon drop.  I guess one is a solution within the card and one is manipulating it from the outside.  Do you have a preference?